include/myst/process.h

```diff
@@ -8,6 +8,8 @@
 #include <sys/types.h>
 #include <unistd.h>
 
+#define MYST_DEFAULT_UMASK (S_IWGRP | S_IWOTH)
```

Comment:
Why do we need permission `S_IWOTH`?

Reply:
This is is the "typical default value" as defined by the umask manpage.

```
 typical default value for the process  umask  is  S_IWGRP | S_IWOTH
       (octal  022).
```